src/main/java/appeng/me/GridNode.java

```diff
@@ -603,7 +604,7 @@ public void setControllerRoute( IPathItem fast, boolean zeroOut )
 	{
 		if( zeroOut )
 		{
-			this.channelData &= ~0xff;
+			this.usedChannels &= ~0xffffffff;
```

Comment:
I just noticed, that every call to `setControllerRoute` is having `zeroOut = true`